app/src/main/java/org/openimis/imispolicies/ClientAndroidInterface.java

```diff
@@ -4211,7 +4211,7 @@ public JSONArray getLanguage() {
     @SuppressWarnings("unused")
     public int getTotalFamily() {
         @Language("SQL")
-        String FamilyQuery = "SELECT count(1) Families  FROM  tblfamilies WHERE isoffline = 1 OR isoffline = 0"; // WHERE isoffline = 1 OR isoffline = 0
+        String FamilyQuery = "SELECT count(1) Families  FROM  tblfamilies WHERE isOffline = 1 or isOffline = 0 or isOffline = \"false\" or isOffline = \"true\""; // WHERE isoffline = 1 OR isoffline = 0
```

Comment:
that is the point of the select ? do you just want to filter out "isoffline is null" ?